### PR TITLE
CORTX-28982 CORTX images generation with RGW support on Rocky Linux 8.4 for community

### PR DESCRIPTION
**Pull request by AbhijitPatil1992**
### Changes
 - [x] Why is this change required? What problem does it solve?
 - Rocky linux deployment changes outside seagate network with rgw docker image.
 - [x] If proposing a new change then please raise an issue first
 - https://jts.seagate.com/browse/CORTX-28982

### How Has This Been Tested? (Optional)
 - [x] Please describe in detail how you tested your changes.
 - [x] Include details of your testing environment, and the tests you ran to
 - [x] How your change affects other areas of the code, etc.

### Checklist
 - [x] tested locally
 - [x] added new dependencies
 - [x] updated the docs
 - [x] added a test

-----
[View rendered doc/community-build/docker/cortx-all/README.md](https://github.com/AbhijitPatil1992/cortx/blob/CORTX-28982/doc/community-build/docker/cortx-all/README.md)

### CODE REVIEW

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]
Please change the title of the page from `Generate CORTX-ALL container image` to `Generate CORTX container images`
We can map /root/rpmbuild to external volume instead of increasing docker home size. Let's take it as an improvement.

**Comment by AbhijitPatil1992** [comment]
> Please change the title of the page from `Generate CORTX-ALL container image` to `Generate CORTX container images` We can map /root/rpmbuild to external volume instead of increasing docker home size. Let's take it as an improvement.

Done required changes and I will make volume related changes as improvements.

File: doc/community-build/docker/cortx-all/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Generate CORTX-ALL container image
+# Generate CORTX container image
```
Comment (shailesh-vaidya) [inline doc/community-build/docker/cortx-all/README.md:1]: Please `s/image/images` as we need two images now.

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]

File: doc/community-build/docker/cortx-all/README.md (hunk 1 above)
Comment (AbhijitPatil1992) [reply doc/community-build/docker/cortx-all/README.md]: Yes just now replaced this as well.

**Review by shailesh-vaidya — APPROVED** [review]

**Review by mukul-seagate11 — APPROVED** [review]
looks good

**Review by johnbent — APPROVED** [review]

**Pull request merged**